website/pages/en/index.js

```diff
@@ -0,0 +1,213 @@
+  return siteConfig.baseUrl + (language ? `${language}/` : '') + page;
+}
+
+class Button extends React.Component {
+  render() {
+    return (
+      <div className="pluginWrapper buttonWrapper">
+        <a className="button" href={this.props.href} target={this.props.target}>
+          {this.props.children}
+        </a>
+      </div>
+    );
+  }
+}
+
+Button.defaultProps = {
+  target: '_self',
+};
+
+const SplashContainer = props => (
+  <div className="homeContainer">
+    <div className="homeSplashFade">
+      <div className="wrapper homeWrapper">{props.children}</div>
+    </div>
+  </div>
+);
+
+const Logo = props => (
+  <div className="projectLogo">
+    <img src={props.img_src} alt="Project Logo" />
+  </div>
+);
+
+const ProjectTitle = () => (
+  <h2 className="projectTitle">
+    {siteConfig.title}
+    <small>{siteConfig.tagline}</small>
+  </h2>
+);
+
+const PromoSection = props => (
+  <div className="section promoSection">
+    <div className="promoRow">
+      <div className="pluginRowBlock">{props.children}</div>
+    </div>
+  </div>
+);
+
+class HomeSplash extends React.Component {
+  render() {
+    const language = this.props.language || '';
+    return (
+      <SplashContainer>
+        <Logo img_src={imgUrl('docusaurus.svg')} />
+        <div className="inner">
+          <ProjectTitle />
+          <PromoSection>
+            <Button href={docUrl('introduction.html', language)}>Get started</Button>
+          </PromoSection>
+        </div>
+      </SplashContainer>
+    );
+  }
+}
+
+const Block = props => (
+  <Container
+    padding={['bottom', 'top']}
+    id={props.id}
+    background={props.background}>
+    <GridBlock align="center" contents={props.children} layout={props.layout} />
+  </Container>
+);
+
+const Features = () => (
+  <Block layout="fourColumn">
+    {[
+      {
+        content: 'This is the content of my feature',
+        image: imgUrl('docusaurus.svg'),
+        imageAlign: 'top',
+        title: 'Feature One',
+      },
+      {
+        content: 'The content of my second feature',
+        image: imgUrl('docusaurus.svg'),
+        imageAlign: 'top',
+        title: 'Feature Two',
+      },
+    ]}
+  </Block>
+);
+
+const FeatureCallout = () => (
+  <div
+    className="productShowcaseSection paddingBottom"
+    style={{textAlign: 'center'}}>
+    <h2>Feature Callout</h2>
+    <MarkdownBlock>These are features of this project</MarkdownBlock>
+  </div>
+);
+
+const LearnHow = () => (
+  <Block background="light">
+    {[
+      {
+        content: 'Talk about learning how to use this',
+        image: imgUrl('docusaurus.svg'),
+        imageAlign: 'right',
+        title: 'Learn How',
+      },
+    ]}
+  </Block>
+);
+
+const TryOut = () => (
+  <Block id="try">
+    {[
+      {
+        content: 'Talk about trying this out',
+        image: imgUrl('docusaurus.svg'),
+        imageAlign: 'left',
+        title: 'Try it Out',
+      },
+    ]}
+  </Block>
+);
+
+const Description = () => (
```

Comment:
I think most of the content that currently lives in "Introduction" should be move to the landing page. This block is a good place where to put it.

Reply:
I'm personally not too happy with the contents in the current introduction. I would prefer to write new stuff. The main points I'd want to highlight are

- saves you time and energy. Focus on code instead of formatting.
- works with many editors and build tools
- simple to enforce in CI